h2o-core/src/main/java/water/fvec/Vec.java

```diff
@@ -320,6 +320,16 @@ public boolean isBinary(){
     return rs._isInt && rs._mins[0] >= 0 && rs._maxs[0] <= 1;
   }
 
+  /**
+   * Check binary vector case -1/1
+   * @return true if the vector is consist of -1/1 only
+   */
+  public boolean isBinaryOnes(){
```

Comment:
it looks like a constant -1 or +1 columns would be also considered binary - is that intended?

Reply:
Good point. If it is not intended, we should also change `isBinary()` method...

Reply:
I wouldn't change isBinary method without evaluating the impact, there is also no test for neither of the modified methods

for core classes there should always be a unit test